tools/devctr/Dockerfile.x86_64

```diff
@@ -94,5 +82,7 @@ RUN mkdir "$TMP_BUILD_DIR" \
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION_TAG}/tini-static-amd64 /sbin/tini
 RUN chmod +x /sbin/tini
 
+RUN python3 -m pip freeze > /tmp/requirements-freeze.txt
```

Comment:
I am wondering why are we getting the freezed versions from the container. Shouldn't they be identical to the current `tools/requirements-freeze.txt`?

Reply:
I think we should have here an update command before freezing the packages, right?

Reply:
> I am wondering why are we getting the freezed versions from the container. Shouldn't they be identical to the current `tools/requirements-freeze.txt`?

As discussed in another comment, the list of installed packages might be bigger than list of base packages to install.

Reply:
> I think we should have here an update command before freezing the packages, right?

Hm..didn't actually follow. 
Line 68 ```RUN python3 -m pip install --requirement /tmp/requirements-freeze.txt ``` reruns package installations every-time docker image is build.

Reply:
I was under the impression that the update script should also run an upgrade command